### PR TITLE
[WIP] Add Connections.Response request type and tests

### PR DESCRIPTION
**Pull request by chrl**
In this PR I wanted to add "Connections.Response" request type, that alexa sends during ISP workflow. Without that request validation fails.

### CODE REVIEW

**Comment by andremaha** [comment]
That would be so awesome, now that ISP came to Europe finally. Many skill developers in PHP and related platforms would appreciate this one!

File: test/Test/Request/RequestData/connectionsResponseRequest.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          "AudioPlayer": {}
+        }
+      }
+    },
+    "AudioPlayer": {
+      "offsetInMilliseconds": 0,
+      "playerActivity": "IDLE"
+    }
+  },
+  "request": {
```
Comment (maxbeckers) [inline test/Test/Request/RequestData/connectionsResponseRequest.json]: The example request looks like this. Please use this structure for the test. And implement the missing parts in `ConnectionsResponseRequest`. For mode details see https://developer.amazon.com/de/docs/in-skill-purchase/add-isps-to-a-skill.html#handle-results
```
{
    "type": "Connections.Response",
    "requestId": "string",
    "timestamp": "string",
    "name": "Upsell",
    "status": {
        "code": "string",
        "message": "string"  
    },
    "payload": {
        "purchaseResult":"ACCEPTED",    
        "productId":"string",   
        "message":"optional additional message"
    },
    "token": "string"
}
```

Comment (chrl) [reply test/Test/Request/RequestData/connectionsResponseRequest.json]: Great, I will do that.

Comment (maxbeckers) [reply test/Test/Request/RequestData/connectionsResponseRequest.json]: Still in progress? Or do you need some help?

**Comment by maxbeckers** [comment]
Will be fixed with #77. Branch was merged to feature.

**Pull request closed without merging**